src/pages/validated-patterns/[...tutorialSlug]/index.tsx

```diff
@@ -21,14 +21,17 @@ export async function getStaticProps({
 }: GetStaticPropsContext<{ tutorialSlug: [string, string] }>): Promise<
 	{ props: ValidatedPatternsTutorialViewProps } | { notFound: boolean }
 > {
-	return { notFound: true }
-	const props = await getValidatedPatternsTutorialViewProps(params.tutorialSlug)
+	try {
+		const props = await getValidatedPatternsTutorialViewProps(params.tutorialSlug)
 
 	// If the tutorial doesn't exist, hit the 404
-	if (!props) {
+		if (!props) {
```

Comment:
@im2nguyen let's log the error here, and also in the catch. Because these two shouldn't fail very often considering the use of `getStaticPaths`.

Reply:
just updated! 

this shouldn't fail at all (unless the directory doesn't exist in `/tutorials`, which is the state we're at right now)